microservices/product-search/bin/console

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env php
```

Comment:
It would be nice if Symfony skeleton could be presented in a separate commit, so we know exactly what has changed by microservice